### PR TITLE
Add Metrics.Net IMetricsClient

### PR DESCRIPTION
**Pull request by tonymobster**
Adds Metrics.Net support (http://www.nuget.org/packages/Metrics.NET/0.2.16). I
find it more feature complete and it supports multiple metrics backends.

Note: It needs to be configured on startup

// Metric.Config
// .WithHttpEndpoint("http://localhost:1234/metrics/")
// .WithAllCounters()
// .WithInternalMetrics()
// .WithReporting(config => config
//     .WithConsoleReport(TimeSpan.FromSeconds(30))
// .WithCSVReports(@"c:\temp\reports\", TimeSpan.FromSeconds(10))
// .WithTextFileReport(@"C:\temp\reports\metrics.txt",
TimeSpan.FromSeconds(10))
// .WithGraphite(new Uri("net.udp://localhost:2003"),
TimeSpan.FromSeconds(1))
// .WithInfluxDb("192.168.1.8", 8086, "admin", "admin", "metrics",
TimeSpan.FromSeconds(1))
// .WithElasticSearch("192.168.1.8", 9200, "metrics",
TimeSpan.FromSeconds(1))
// );

### CODE REVIEW

File: src/AzureStorage/Foundatio.AzureStorage.csproj [hunk 1]
```diff
@@ -75,6 +75,7 @@
     <Compile Include="Storage\AzureFileStorage.cs" />
   </ItemGroup>
   <ItemGroup>
+    <None Include="app.config" />
```
Comment (niemyjski) [inline src/AzureStorage/Foundatio.AzureStorage.csproj]: Can you revert this change

File: src/AzureStorage/app.config [hunk 2]
```diff
@@ -0,0 +1,23 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
```
Comment (niemyjski) [inline src/AzureStorage/app.config]: Can you revert these changes to the app.config

**Comment by niemyjski** [comment]
Thanks for your changes. I just noticed it was in the root folder and just did a commit to move it into it's own folder.

**Comment by tonymobster** [comment]
I actually just noticed that myself. Was gonna mention it. 
Welcome. Glad to help. Very useful library.

**Comment by niemyjski** [comment]
I just renamed the metrics client with the proper casing :). It will be available here: https://www.nuget.org/packages/Foundatio.MetricsNET/
